src/main/java/org/jd/tests/App.java

```diff
@@ -9,5 +9,6 @@
     public static void main( String[] args )
     {
         System.out.println( "Hello World!" );
+        System.out.println( "Hello World!" );
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this use of System.out or System.err by a logger. ([squid:S106](http://localhost:9000/coding_rules#rule_key=squid%3AS106))

[See it in SonarQube](http://localhost:9000/project/issues?id=org.jd.tests%3Atest-project&pullRequest=1&issues=AWlY4UlWcP19gDvLMZq6&open=AWlY4UlWcP19gDvLMZq6)

[ISSUE_KEY]: AWlY4UlWcP19gDvLMZq6